CHANGELOG

```diff
@@ -1,6 +1,8 @@
 # 3.15.0 (2024-XX-XX)
 
- * Deprecate using `~` with `+` or `-` in an expression without using parentheses to clarify precedence
+ * Deprecate using the `not` unary operator in an expression with ``*``, ``/``, ``//``, or ``%`` without using explicit parentheses to clarify precedence
+ * Deprecate using the `??` binary operator without explicit parentheses
+ * Deprecate using the `~` binary operator in an expression with `+` or `-` without using parentheses to clarify precedence
```

Comment:
this should also mention `Add support for triggering deprecations for future operator precedence changes` (and maybe this should have docymentation as well)